build/run-docker-tests.sh

```diff
@@ -54,6 +53,11 @@ export PAUSE=${PAUSE:-60}
 echo sleep $PAUSE seconds cypress ...
 sleep $PAUSE
 export CI=true # force cypress to output color
+# Check for standalone test suite flag to loosen tests if there are other policies present from concurrent tests
+if [ "${STANDALONE_TESTSUITE_EXECUTION}" == "false" ]; then
```

Comment:
I am afraid that this patch won't make a difference. IMHO the right place to do the change suggested in the previous comment is
https://github.com/open-cluster-management/grc-ui/blob/ce3c9411eb30ea87bc8e0c6cf24dd0e1c4bbc054/build/run-e2e-tests.sh#L52

Reply:
You are correct that we'd need to put the variable in the Docker env. However, `run-e2e-tests.sh` is only called by this repo, and the `docker` command you're referencing is pulling the latest API image to use alongside the UI: https://github.com/open-cluster-management/grc-ui/blob/ce3c9411eb30ea87bc8e0c6cf24dd0e1c4bbc054/build/run-e2e-tests.sh#L49
`Dockerfile.e2etest` is the containerized version of our tests, which pulls in `run-docker-tests.sh` for test execution, which is why it needs to be set there. We call the `docker` command in the `governance-policy-framework` tests, which is where the variable is set in the Docker env: https://github.com/open-cluster-management/governance-policy-framework/blob/b68ed6365c7e37d3c87fff82c74e449755d68148/build/run-e2e-tests-ui.sh#L40-L50

Reply:
I see. That makes sense. Thank you.

Reply:
Anyway, if you would define CYPRESS_ prefixed variable directly in `docker - - env` you wouldn't have to redefine it again in the script.

Reply:
Ah, you're right--I was following the FAIL_FAST pattern, but that doesn't make sense here since we don't need customized behaviors for this variable like we do with FAIL_FAST. Let's make sure everything flows correctly, then I've submitted a PR to fix it downstream: https://github.com/open-cluster-management/governance-policy-framework/pull/80 If that works, we can remove it here in a future PR.